server/app/services/applicant/ApplicantService.java

```diff
@@ -284,7 +286,13 @@ public CompletionStage<ReadOnlyApplicantProgramService> stageAndUpdateIfValid(
     }
 
     return stageAndUpdateIfValid(
-        applicantId, programId, blockId, updateMap, updates, addressServiceAreaValidationEnabled);
+        applicantId,
+        programId,
+        blockId,
+        updateMap,
+        updates,
+        addressServiceAreaValidationEnabled,
+        forceUpdate);
```

Comment:
Thought (non-blocking): this PR is quite large and it would have been nice to split it up, for example a separate PR to add support for forcing updates. And maybe frontend vs backend parts or whatever other splits are reasonable.

Actually the more I think about it the more I prefer that you split it up. But I'll leave it up to you.

Reply:
I don't really like splitting the force update change since then you end up submitting code that isn't used by anything else, without context. (Or alternatively, if you submit removeFile first, it's submitting code that doesn't actually work.) 

I don't think splitting the frontend and backend is worth the effort at this point, but agree it should have been split.

Reply:
I disagree pretty strongly on the first point - mentioning in the PR description that it is part of a multi-PR change, and linking all related PRs in a bug, solves that problem from my perspective.